src/main/java/org/generationcp/middleware/pojos/Germplasm.java

```diff
@@ -62,7 +62,7 @@
 	@NamedQuery(name = "getAllGermplasm", query = "FROM Germplasm"),
 	@NamedQuery(name = "countAllGermplasm", query = "SELECT COUNT(g) FROM Germplasm g"),
 
-	@NamedQuery(name = "countMatchGermplasmInList", query = "SELECT COUNT(g) FROM Germplasm g WHERE g.gid IN (:gids)")
+	@NamedQuery(name = "countMatchGermplasmInList", query = "SELECT COUNT(g) FROM Germplasm g WHERE g.gid IN (:gids) AND g.deleted = 0")
```

Comment:
Hi @cheryleigh 
The call for this query is called in GermplasmDataManagerImpl. Move it to GermplasmService and call it from there. We're slowly deprecating Data Managers and moving the methods to the Service classes.